### PR TITLE
add package py-jacobi

### PR DESCRIPTION
**Pull request by jonas-eschle**
Add the package py-jacobi

### CODE REVIEW

**Comment by jonas-eschle** [comment]
@spackbot fix style

**Comment by jonas-eschle** [comment]
Ready from my side

File: var/spack/repos/builtin/packages/py-jacobi/package.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    maintainers("jonas-eschle")
+    license("MIT", checked_by="jonas-eschle")
+
+    version("0.9.2", sha256="c11f481663246ef1c2da915b9f9ab4ef229051fb14e0afc232d4668301320828")
+
+    depends_on("python@3.8:", type=("build", "run"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-toml", type=("build"))
+    depends_on("py-setuptools", type="build")
+    depends_on("py-numpy", type="run")
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-jacobi/package.py]:
```suggestion
    depends_on("py-setuptools@42:", type="build")
    depends_on("py-setuptools-scm@3.4:+toml", type="build")
    depends_on("py-numpy", type=("build", "run"))
```

**Review by adamjstewart — CHANGES_REQUESTED** [review]

**Comment by jonas-eschle** [comment]
Thanks for the fix, that is indeed the right dependency with setuptools-scm

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**